pipeline/4/Jenkinsfile-tier-0.groovy

```diff
@@ -60,7 +34,45 @@ node(nodeName) {
     }
 
     timeout(unit: "HOURS", time: 6) {
-        parallel testStages
+	    stage('RPM Sanity suite RHEL-8') {
+                try {
+		script {
+		    withEnv([
+			"osVersion=RHEL-8",
+			"sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
+			"sutConf=conf/${cephVersion}/ansible/sanity-ceph-ansible.yaml",
+			"testSuite=suites/${cephVersion}/ansible/sanity_ceph_ansible.yaml",
+                        "containerized=false",
+			"addnArgs=--post-results --log-level DEBUG"
+		    ]) {
+			sharedLib.runTestSuite()
+		    }
+		}
+                }
+                catch(Exception ex){ 
+                     echo 'Exception occured' 
+                     echo ex.getMessage() 
+                } 
+	    }
+	    stage('Containerized Sanity suite RHEL-8') {
```

Comment:
We could rename the stage as `Image sanity`

Reply:
Since we cannot rename the file as mentioned in the previous comment, should i still change this or leave it as is?